src/main/java/javaserver/Responders/FormResponder.java

```diff
@@ -0,0 +1,45 @@
+package javaserver.Responders;
+
+import javaserver.Form;
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+
+public class FormResponder implements Responder {
+
+	private String[] supportedMethods;
+	private Form form;
+
+	public FormResponder(String[] supportedMethods, Form form) {
+		this.supportedMethods = supportedMethods;
+		this.form = form;
+	}
+
+	@Override
+	public Response getResponse(Request request) {
+		return new Response.ResponseBuilder(getStatusLine(request))
+      .body(getData(request))
```

Comment:
@jayden It's strange, Vim makes it seem like these lines are indented correctly but Github and Eclipse have the weird indentation. Any idea what's going on? 
![screen shot 2016-04-07 at 10 10 06 am](https://cloud.githubusercontent.com/assets/10511338/14359996/fa85ceec-fca8-11e5-9b24-c5d4adb16271.png)